docs/xshareclasses.md

```diff
@@ -437,7 +437,28 @@ behavior, which can improve the performance of class loading from the shared cla
 
 :   Uses a persistent cache. The cache is created on disk, which persists beyond operating system restarts. Non-persistent and persistent caches can have the same name. On AIX, you must always use the `persistent` suboption when you run utilities such as `destroy` on a persistent cache.
 
-:   :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** Persisent caches are not supported on z/OS systems. z/OS supports only non-persistent caches.
+:   :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Notes:**
+
+:    - On 31-bit VM on all versions of z/OS, the shared cache is memory mapped within the 0-2 GB address range. The size of the persistent shared class cache is limited by the system limit `MAXMMAPAREA`. For more information about the suggested value for the `MAXMMAPAREA` limit, see [Setting resource limits (z/OS)](https://www.eclipse.org/openj9/docs/configuring/#setting-resource-limits-zos).
+     - On 64-bit VM, the shared cache is mapped as follows:
+         - On z/OS version 2.3 and earlier, the persistent shared cache is memory mapped within the 0-2 GB address range. The size of the persistent shared class cache is limited by the system limit `MAXMMAPAREA`.
```

Comment:
> The size of the persistent shared class cache

and in the next paragraph:
> The size of persistent shared class cache is limited by the `MAXSHARE` value

Better to say "The maximum size of persistent shared class cache".